libavfilter/dnn/dnn_backend_tf.c

```diff
@@ -723,6 +754,7 @@ DNNModel *ff_dnn_load_model_tf(const char *model_filename, DNNFunctionType func_
         }
     }
 
+    tf_model->inference_queue = ff_queue_create();
```

Comment:
if (!tf_model->inference_queue) {
free...
return NULL;
}

on other comment for the first patch.

Reply:
Yes, the error handling of this part is in the commit "lavfi/dnn_backend_tf: Error Handling".

Reply:
ok, so no problem of the first patch.